packages/react-native-editor/babel.config.js

```diff
@@ -47,7 +47,7 @@ module.exports = function ( api ) {
 						},
 					],
 				],
-				exclude: /node_modules\/(react-native|@react-native-community|@react-navigation)/,
+				exclude: /node_modules\/(react-native|@react-native-community|@react-navigation|@sentry)/,
```

Comment:
It doesn't feel right that we need to include `@sentry` here since, if I understand correctly, the sentry package is installed in gutenberg-mobile and not Gutenberg. Is there any way we could handle this in gutenberg-mobile instead of here?

Reply:
Right, although this is the file we use in the Babel configuration on gutenberg-mobile:
https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/babel.config.js

I agree that doesn't make sense to keep a reference to Sentry in a repo where the package doesn't exist, so I'm going to duplicate the content of this file on gutenberg-mobile and leave this file untouched 👍 .

Reply:
Now that you mention it, I'm going to double-check if we need to exclude Sentry here because I added this change in the first commits and since I disabled most of the integrations it might be no longer necessary.

Reply:
> Now that you mention it, I'm going to double-check if we need to exclude Sentry here because I added this change in the first commits and since I disabled most of the integrations it might be no longer necessary.

Yep, I confirm that it's no longer required 🎊 , I've just reverted this change in [this commit](https://github.com/WordPress/gutenberg/pull/32768/commits/ac4b155f3c944b90b319d1c707f61af1736772bd).